pull_datasets/mp_redshift.yaml

```
@@ -0,0 +1,6 @@
+ name: mp_redshift
```

```suggestion
 name: mp-redshift
```
Will need to be a dash to deploy